### PR TITLE
Kindd: remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by LinArcX**
1. Let users [decide](https://github.com/LinArcX/kindd/#polkit) which polkit-agent is appropriate for them.
2. Remove unnecessary runtime dependencies like:
`polkit-gnome` , `qt5-quickcontrols2`, `qt5-declarative`, `qt5-svg`

### CODE REVIEW

**Comment by Piraty** [comment]
lol this seems to be your very project and it lists `qt5-svg` as runtime dependency, yet you remove it from the template?

When this GUI is for those who don't want to `sudo dd`, do you really think the intended target audience is supposed to "decide which polkit-agent is appropriate for them"?

**Comment by LinArcX** [comment]
> yet you remove it from the template?
Yes. 

> When this GUI is for those who don't want to sudo dd..
No. It's not the main idea behind kindd. The idea is providing a safe place to create bootable USB from iso files. Why polkit agent should be optional? Imagine you've installed pantheon, ad it already will install pantheon agent polkit. When you install kindd, it'll install gnome agent polkit too. Why there should be two diffeent agents on your machine?

So, I think it's better to let users to decide install which agent polkit.

**Comment by not-chicken** [comment]
While `qt5-quickcontrols2`, `qt5-declarative` are automatically detected by xbps as runtime deps, `qt5-svg` is not. Removing it will break icon support.
kindd

I'd hate to not see that lovely mascot while using `kindd`.

**Comment by LinArcX** [comment]
> ... detected by xbps-src as runtime deps, qt5-svg is not.

Why?

**Comment by not-chicken** [comment]
> > ... detected by xbps-src as runtime deps, qt5-svg is not.
> 
> Why?

Kindd links with the shlibs, aka shared libraries provided by the other two packages, so xbps-src detects that those packages must be installed to load those libraries at runtime.

It doesn't need to link against any of `qt5-svg` shlibs but uses the package to render the svg icons. So you have to mention it as a runtime dependency.

**Comment by LinArcX** [comment]
> While qt5-quickcontrols2, qt5-declarative are automatically detected by xbps-src as runtime deps, qt5-svg is not.

I want to know exactly why this didn't happen?

Also I've noticed that i had `qt5-svg` on my machine (as dependency of other packages). So removing `qt5-svg` didn't affect my experience with software.(Sorry about that!)

Still I'm confused! Why some `makedepends` detected as runtime dependencies, and other don't.

**Comment by LinArcX** [comment]
@not-chicken Thank you for explanations.

I've noticed that i had `qt5-svg` on my machine (as dependency of other packages). So removing `qt5-svg` didn't affect my experience with software.(Sorry about that!)

So I added it again.

**Comment by not-chicken** [comment]
> I've noticed that i had `qt5-svg` on my machine (as dependency of other packages). So removing `qt5-svg` didn't affect my experience with software.(Sorry about that!)

 Yeah it's a common dep of qt5 apps. I did the same mistake, see #20608 
I'd recommend testing packages in a vm or in a secondary installation with only base-system, xorg and a wm. It will make it easy to catch missing runtime dependencies.

File: srcpkgs/kindd/template [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Template file for 'kindd'
 pkgname=kindd
 version=2.1.0
-revision=1
+revision=2
 build_style=cmake
 hostmakedepends="cmake pkg-config qt5-qmake qt5-host-tools"
```
Comment (Piraty) [inline srcpkgs/kindd/template:6]: adding `cmake` to `hostmakedepends` is redundant when `build_style=cmake`

**Pull request merged**